packages/tools/src/tools/annotation/SplineROITool.ts

```diff
@@ -121,6 +120,18 @@ class SplineROITool extends ContourSegmentationBaseTool {
          * modifier must be pressed when the first point of a new contour is added.
          */
         contourHoleAdditionModifierKey: KeyboardBindings.Shift,
+        /**
+         * The polyline may get processed in order to reduce the number of points
+         * for better performance and storage.
+         */
+        decimate: {
+          enabled: true,
```

Comment:
I prefer enabled to be false as default for now for all tools

Reply:
Updated